Getting error from notificationBuilder.addAction()

C:\Users\siva_anish\Desktop\foreground RN\AwesomeProject\node_modules\@supersami\rn-foreground-service\android\src\main\java\com\supersami\foregroundservice\NotificationHelper.java:144: error: cannot find symbol
            notificationBuilder.addAction(R.drawable.redbox_top_border_background, bundle.getString("buttonText", "Button"), pendingBtnIntent);
                                           ^
  symbol:   variable drawable
  location: class R
C:\Users\siva_anish\Desktop\foreground RN\AwesomeProject\node_modules\@supersami\rn-foreground-service\android\src\main\java\com\supersami\foregroundservice\NotificationHelper.java:148: error: cannot find symbol
            notificationBuilder.addAction(R.drawable.redbox_top_border_background, bundle.getString("button2Text", "Button"), pendingBtn2Intent);
                                           ^
  symbol:   variable drawable
  location: class R
Note: C:\Users\siva_anish\Desktop\foreground RN\AwesomeProject\node_modules\@supersami\rn-foreground-service\android\src\main\java\com\supersami\foregroundservice\ForegroundService.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
2 errors

FAILURE: Build failed with an exception.


code // 

   if(bundle.getBoolean("button", false) == true){
            notificationBuilder.addAction(R.drawable.redbox_top_border_background, bundle.getString("buttonText", "Button"), pendingBtnIntent);
        }

        if(bundle.getBoolean("button2", false) == true){
            notificationBuilder.addAction(R.drawable.redbox_top_border_background, bundle.getString("button2Text", "Button"), pendingBtn2Intent);
        }

--- a/android/src/main/java/com/supersami/foregroundservice/NotificationHelper.java
+++ b/android/src/main/java/com/supersami/foregroundservice/NotificationHelper.java
@@ -14,6 +14,7 @@ import android.os.Bundle;
 import androidx.core.app.NotificationCompat;
 import android.util.Log;
 
+import com.facebook.react.R;
 
 // partially took ideas from: https://github.com/zo0r/react-native-push-notification/blob/master/android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotificationHelper.java
 

